apps/backend/src/validators/validationSchemas.ts

```diff
@@ -0,0 +1,9 @@
+import Joi from 'joi';
+
+export const registerSchema = Joi.object({
```

Comment:
validationSchema must have the same types and lenghts as SQL schema and Prisma model.